consensus/src/quorum_store/tests/utils.rs

```diff
@@ -381,7 +381,7 @@ fn test_proof_calculate_remaining_txns_and_proofs() {
 #[test]
 fn test_proof_pull_proofs_with_duplicates() {
     let my_peer_id = PeerId::random();
-    let mut proof_queue = ProofQueue::new(my_peer_id);
+    let mut proof_queue = BatchProofQueue::new(my_peer_id);
```

Comment:
BatchProofQueue::new has 2 arguments?

Reply:
Will fix the tests, just wanted to make sure the logic was sound.